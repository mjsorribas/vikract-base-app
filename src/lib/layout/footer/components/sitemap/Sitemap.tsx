import { Link, Stack } from "@chakra-ui/react";

export default function Sitemap() {
  return (
    <Stack mr={6} ml={6} align="flex-start">
      <Link href="https://vikract.com/aboutus">
        About Vikract
      </Link>
      <Link href="https://vikract.com/roadmap">
        About our Roadmap
      </Link>
      <Link href="https://vikract.com/documentationfordevs">
        Documentation for developers
      </Link>
      <Link href="https://vikract.com/contactus">
        Contact us
      </Link>
    </Stack>
  );
}
