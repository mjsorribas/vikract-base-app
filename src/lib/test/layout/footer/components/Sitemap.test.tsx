import renderer from "react-test-renderer";
import { expect, test } from "vitest";

import Sitemap from "lib/layout/footer/components/sitemap/Sitemap";

const toJson = (component: renderer.ReactTestRenderer) => {
  const result = component.toJSON();
  expect(result).toBeDefined();
  expect(result).not.toBeInstanceOf(Array);
  return result as renderer.ReactTestRendererJSON;
};

test("Sitemap", () => {
  const component = renderer.create(<Sitemap />);
  const tree = toJson(component);
  expect(tree).toMatchSnapshot();
});
