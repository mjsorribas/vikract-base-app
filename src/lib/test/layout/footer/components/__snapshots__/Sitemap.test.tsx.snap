// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Sitemap 1`] = `
<div
  className="chakra-stack css-111p793"
>
  <a
    className="chakra-link css-0"
    href="https://vikract.com/aboutus"
  >
    About Vikract
  </a>
  <a
    className="chakra-link css-0"
    href="https://vikract.com/roadmap"
  >
    About our Roadmap
  </a>
  <a
    className="chakra-link css-0"
    href="https://vikract.com/documentationfordevs"
  >
    Documentation for developers
  </a>
  <a
    className="chakra-link css-0"
    href="https://vikract.com/contactus"
  >
    Contact us
  </a>
</div>
`;
